storage/ResidentDatabaseSchema.ts

```diff
@@ -15,7 +15,10 @@ type ResidentDatabaseSchema = NamedSchema<
       value: {
         type: string;
         images: string[];
-        notes: string;
+        notes: {
+          value: string;
+          isPostVisitAction: boolean;
+        };
```

Comment:
We'll need a migration for these, too. Remember, they're in a different database from the others.